### DISCUSSION TITLE
Car in front of the door

### DISCUSSION THREAD

**Discussion by lorenzomartini1** [blakeblackshear/frigate#7233] [Ask A Question]
How is this possible to detect a car in front of the door? It doesn't even look like it...
![image](https://github.com/blakeblackshear/frigate/assets/35630216/9b00d839-c930-4aa9-9a61-c54c94c8036f)

**Comment by NickM-27** [comment]
The built in models are demo models from Google that are trained on the COCO dataset which is not trained on images from security cameras. This means it has false positives due to the difference in types of images used. 

Here are some examples of images from the COCO dataset: https://imgur.com/a/CNImofJ

You can see more here: https://cocodataset.org/#explore

**Comment by matt-snelling** [comment]
>It doesn't even look like it... 

To you, of course not!  I can't argue with you there.  But if you're interested, it's worth reading up on computer vision - it's ridiculously complicated and based entirely on maths - as computers don't have eyes!

Check out the [Frigate+](https://plus.frigate.video) project - which is the upcoming attempt at improving things. I believe you can submit this picture, and "verify and save" without labelling anything to effectively train the upcoming model that this was a false positive - like this:

![image](https://github.com/blakeblackshear/frigate/assets/14086708/99b2267b-3327-4046-9c58-5bb1ee025117)
